scripts/templates/field_details.j2

```diff
@@ -4,6 +4,12 @@
 
 {{ fieldset['description']|replace("\n", "\n\n") }}
 
+{%- if usage_doc %}
```

Comment:
I don't believe this PR's branch here includes the changes made in #1354, so this change is trying to revert it by adding this here and removing https://github.com/elastic/ecs/pull/1366/files#diff-e2eb14d7048aba757706fb174c9a191a774f83bbeaf7bd25f9148e7ed56603f7L144.

Reply:
I think I got this sorted now. Not quite sure what I did there